src/operator/optimizer_op.cu

```diff
@@ -51,5 +51,9 @@ NNVM_REGISTER_OP(rmsprop_update)
 NNVM_REGISTER_OP(rmspropalex_update)
 .set_attr<FCompute>("FCompute<gpu>", RMSPropAlexUpdate<gpu>);
 
+NNVM_REGISTER_OP(ftrl_update)
+.set_attr<FCompute>("FCompute<gpu>", FtrlUpdate<gpu>)
+.set_attr<FComputeEx>("FComputeEx<gpu>", FtrlUpdateEx<gpu>);
```

Comment:
Are you suggesting removing `FCompute<gpu>` for Ftrl op? If we remove `FtrlUpdate<gpu>` from the op registration it will break backward compatibility and affect other people who are using it.

Reply:
oh, i know. FCompute\<gpu\> is for default storage. FComputeEx is for non-default storage. tks.